types/multiavatar__multiavatar/test/multiavatar__multiavatar-tests-node.ts

```diff
@@ -0,0 +1,3 @@
+/// <reference types="node" />
+import multiavatarNode = require('@multiavatar/multiavatar');
+const svgCodeGeneratedFromNode = multiavatarNode('Binx Bond');
```

Comment:
can we skip this file? this would allow to skip node dependency, thx!
that would be ok, you're testing other usage

Reply:
I'm not sure I follow.
The node dependency will not be  in the final `package.json` for this published types package, or will it?

Also, this file is testing the usage via node and commons, unlike the other two test files which test usage via ESM and UMD. 

The package itself (officially) supports all three. That's what drove me to write three test files.

Reply:
Yes, the intent is to remove stress on DT repo, put by plethora of packages that reference Node types, eg. only during tests. This harms other efforts to maintain NodeJS types

Reply:
TL;DR; the DT declaration itself (index.d.ts) has no dependency on Node types

Reply:
Got it. Removing this line though means that this test breaks (since `require` is no longer define). Therefore, do you prefer it if I keep this test (along with the dependency on the Node types), or remove it?

Reply:
You can remove test for CommonJS completely, just for this reason, that's OK or you could try old school
`import * as * from ` if it works for this package